grisha.pletin/First/Task2/2.rb

```diff
@@ -0,0 +1,2 @@
+a = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+p a.select {|k| k =~ /\byes\w*/ }.size
```

Comment:
```suggestion
p a.select { |k| k =~ /\byes\w*/ }.size
```

Reply:
Please check your regular expression, this regexp will work for this hash, but it will not work for a hash with a key 'a yesteraday' for example. Your regular expression only checks that "yes" is present inside of the key name ![image](https://user-images.githubusercontent.com/16926726/97109392-a661ac80-16db-11eb-82af-3a0c8879b005.png)